xamarin/Xamarin.Forms
# [Android] Fixes cancel Webview Navigation

### Description of Change ###

Call `Navigating` event before load Url.

### Issues Resolved ### 

- fixes #4891
- fixes #4919
- closes #5460

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Testing Procedure ###

- run UItest4919
- check that the webpage does not load on init
- click on `Go to github` button
- check that the webpage does not load
- click on `Toggle cancel navigation` button
- click on `Go to github` button
- webpage should load
- all links should work
- click on `Toggle cancel navigation` button
- all links should not work

### PR Checklist ###
